doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -705,6 +705,7 @@ Documentation
   * :ref:`app_memory` with sections for Matter and Zigbee.
   * :ref:`thread_ug_feature_updating_libs` to clarify the use, and added |VSC| instructions.
   * :ref:`ug_thread_communication` by moving it to a separate page instead of it being under :ref:`ug_thread_architectures`.
+  * Added notice box to several nRF Cloud samples using the `nRF Cloud REST API`_ indicating the need for valid device certificate with registered public key.
```

Comment:
```suggestion
  * Added a note to several nRF Cloud samples using the `nRF Cloud REST API`_ indicating the need for a valid device certificate with the registered public key.
```

Reply:
Deviated, here, since I should have written "credentials" not "certificate" in the first place